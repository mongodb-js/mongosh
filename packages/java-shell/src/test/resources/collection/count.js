// before
db.coll.deleteMany({});
db.coll.insertOne({a: 1});
db.coll.insertOne({a: 2});
db.coll.insertOne({a: 3});
// command
db.coll.count();
// clear
db.coll.drop();
