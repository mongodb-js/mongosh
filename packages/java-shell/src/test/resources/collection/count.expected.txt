[ { "0": "DeprecationWarning: Collection.count() is deprecated. Use countDocuments or estimatedDocumentCount." } ]
